### ISSUE TITLE
Saving cheats, donating, etc.

### ISSUE THREAD

**Issue by Kecki8472** [OpenEmu/OpenEmu#3511]
Hello everybody!

I'm sorry, but there's quite a bunch of issues I'd like to bring to discussion. If I may be wrong, PLEASE forgive, but, to be honest, I'm not very good at web-based communication.

1) isn't there also a way to get in contact with the OpenEmu-team via e-mail?? I am visually impaired, and especially with Github I have BIG BIG troubles with overview and orientation. I can imagine that you may worry that I may be a "dummy-user" and become a real pain in your neck after knowing an e-mail-address. Don't worry, actually I'm very familiar with emulation in general and with OpenEmu especially. So, you don't need to worry that I'll contact you because of every little and meaningless thing. Probably someone from the team may be so kind to send a short reply to kecki8472@gmail.com...

2) Some time ago I tried to donate something, but also with this site I head BIG difficulties with overview and orientation. Is there a way to make donations anonymously? What about paysafe-cards?? If not, OK, but asking is still for free, isn't it??

3) What about the ability to save cheats? Currently all entered cheat-codes get lost after an OpenEmu-session. Or did I oversee something?? If not, is there the intention to implement a cheat-saving-function??

4) Light-gun-games can be played with the mouse as light-gun-replacement, right?
The problem here is that Mac OS' mousepointer is TERRIBLY hard to see. It's VERY small, and its color is often VERY bad contrasting.
OK, I know, of course it also has to do with visual impairment. But don't you think that others may also have more fun if in a case of light-gun-necessity the mouse pointer becomes a well-contrasting cross-hair??
Or is there an alternative to the mouse as light-gun-replacement? As far as I know a light-gun for PCs does not exist, especially because of the old-fassioned CRT-technology light-guns in former days have been designed for....
However, for any ideas I would be VERY thankful.

For now, all the best and fine greetings
Kecki

**Comment by clobber** [comment]
1. Sorry, for bugs/issues we only do communication via GitHub.

2. There are [various options](http://openemu.org/donate/).

3. Duplicate of #782

4. The only option we can give you right now is Apple > System Preferences > Accessibility > Display > Cursor size.

**Comment by Kecki8472** [comment]
Dear Clobber!

Thanks a lot for this prompt reply.
OK, so far … Somehow I’ll arrange with Github.
And the other stuff,… OK, also so far everything clear.
Especially the way you managed with saving cheats is nice; funny, but nice. When you know how and what to do, it’s OK. But take it as a suggestion to implement in a future-update … OK, forget it. In this thread you referred me you mentioned even a big cheat-DB included with the emu and the option to add and save own…
And for the issue of small mouse-pointers I also found a nice solution. It’s the mousecape-app, that allows the user total mousepointer-customization. It took some time, but meanwhile all necessary stuff I know to get a well-visible pointer for light-gun-games. :))

OK, hope you get/read this, but your mail gave at the end the option also reply directly, what I now did. I thought others do not need to read this more private stuff.
However, should occur something again,… everything else then via the Github-forum….

Once more thanks a lot and best wishes
Kecki
